src/main/webapp/app/lecture/shared/transcript-viewer/transcript-viewer.component.ts

```diff
@@ -3,15 +3,15 @@ import { CommonModule } from '@angular/common';
 import { FormsModule } from '@angular/forms';
 import { FaIconComponent } from '@fortawesome/angular-fontawesome';
 import { faChevronDown, faChevronUp, faSearch, faTimes } from '@fortawesome/free-solid-svg-icons';
-import { TranslateDirective, TranslatePipe } from '@ngx-translate/core';
+import { TranslatePipe } from '@ngx-translate/core';
 import { escapeString } from 'app/shared/util/text.utils';
 
 import { TranscriptSegment } from 'app/lecture/shared/models/transcript-segment.model';
 
 @Component({
     selector: 'jhi-transcript-viewer',
     standalone: true,
-    imports: [CommonModule, FormsModule, FaIconComponent, TranslateDirective, TranslatePipe],
+    imports: [CommonModule, FormsModule, FaIconComponent, TranslatePipe],
```

Comment:
I do not understand that change

1. As I understand our guidelines we want to use `TranslateDirective` over `TranslatePipe`
2. We still have `<div class="transcript-title" jhiTranslate="artemisApp.attachmentVideoUnit.videoPlayer.transcript">Transcript</div>` in the template, so this just wont work anymore, or am I mistaken? 

I think we should rather replace the usages of `[placeholder]="'artemisApp.attachmentVideoUnit.videoPlayer.searchPlaceholder' | translate"`  
I might have gotten the guidelines wrong there 😅

Reply:
but you might be right, I will double check it